apps/ledger-live-desktop/src/renderer/components/RemoteConfig.jsx

```diff
@@ -20,8 +16,8 @@ export type RemoteConfig = {
 
 const defaultValue: RemoteConfig = {
   data: null,
-  error: null,
-  lastUpdatedAt: null,
+  error: new Error("RemoteConfig is no longer used"),
```

Comment:
because it's too risky and impacting to change the rest, I stopped at this level and we need to schedule a rework of the electron update system (because it somehow touch this)